viewer/src/components/SpaceHex.vue

```diff
@@ -157,6 +165,14 @@ svg {
       }
     }
 
+    .highlight-recent-actions &.current-round:not(.highlighted):not(.toSelect) {
+      fill: $current-round;
```

Comment:
Something that can be done now in CSS3 is 

```
fill: var(--current-round);
```

and in the main component, eg Game.vue:

```css
// also add this class to Game
.game {
  --current-round: black;
}
```

Anyway not a change request, since the rest of the code uses scss variables, just something for the future.